### DISCUSSION TITLE
Toolbar

### DISCUSSION THREAD

**Discussion by pauloargenal** [diegomura/react-pdf#1108] [Q&A] [answered]
How to add a close button inside the toolbar? I wanted to add a close button in our PDFViewer  per reading the document it seems i cannot find any option to customize these toolbar and add a close button.
![image](https://user-images.githubusercontent.com/20692938/106542573-09ff3f80-653f-11eb-867a-a0e9646f004d.png)

**Comment by Chagall** [comment, marked as answer]
Correct me if I'm wrong but I think these viewers vary from browser to browser. I don't think there is a way to actually modify them.

**Comment by diegomura** [comment]
That's correct! Thanks @Chagall ! 

For the moment all I can help with is pointing out how the REPL is created if someone needs to apply some custom styling. You can find that [here](https://github.com/react-pdf/site/blob/master/src/components/PDFViewer.js). Version 2.0 will support building these sort of implementations internally, but first we need to come up with a nice API

**Reply by pauloargenal** [reply]
Alright, thanks guys that's good information for me
